### ISSUE TITLE
Using KiCAD repository symbols/footprints

### ISSUE THREAD

**Issue by ki5libs** [daveshah1/TrellisBoard#7]
Hello @daveshah1 

Nice board! But why do you use local libraries? Some symbols (ex. 93AA46B_OT in eco_memory) exist in upstream KiCAD repository (93AAxxB in Memory_EEPROM). What do you think about my help for creating  KiCAD repository PRs for the rest of the symbols (and may be footprints) with some reworkings in reasonable cases? It would be a good idea to use symbols from the repository in rev.1.1, woudn't it?

Best regards,
Anton Lysak

**Comment by daveshah1** [comment]
In most cases, local libraries were used when I couldn't find a suitable symbol in the included ones, in some cases this might have been historical and an equivalent symbol is now upstream. I would certainly welcome your help in upstreaming symbols and move towards using upstreamed symbols wherever possible.

The ECP5 symbol came from a generator (https://github.com/xesscorp/KiPart/), I'm not sure how that would work in terms of upstreaming - given the large number of pins, one would want to avoid significant manual fixes.

**Comment by ki5libs** [comment]
Perfect! Do you have any BOM for producing this board, which could be used as a datasheet source? Just because any symbol in the KiCAD repository should have the datasheet link.

**Comment by daveshah1** [comment]
Yes, https://github.com/daveshah1/TrellisBoard/blob/master/hardware/ecp5_mainboard/mainboard_prod/ecp5_mainboard_bom_elecrow.xls (generating more of the BOM automatically might be something to look at in the future)
